vnext/Shared/OInstance.cpp

```diff
@@ -508,7 +508,7 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       }
     } else {
 #if (defined(_MSC_VER) && !defined(WINRT))
-      std::string bundlePath = (fs::u8path(m_devSettings->bundleRootPath) / jsBundleRelativePath).string();
+      std::string bundlePath = (fs::u8path(m_devSettings->bundleRootPath) / jsBundleRelativePath).generic_u8string();
```

Comment:
@rozele I think you want `.u8string()` instead? any reason to use the `generic_u8string()` instead?

Reply:
Nope. u8string is better :)